src/app/games/tablut/tests/tablut.component.spec.ts

```diff
@@ -26,8 +27,7 @@ describe('TablutComponent', () => {
         await componentTestUtils.expectClickFailure('#click_4_4', RulesFailure.CANNOT_CHOOSE_ENEMY_PIECE);
     }));
     it('Should cancel move when first click on empty case', fakeAsync( async() => {
-        const message: string = 'Pour votre premier clic, choisissez une de vos pièces.';
-        await componentTestUtils.expectClickFailure('#click_0_0', message);
+        await componentTestUtils.expectClickFailure('#click_0_0', TablutFailure.CHOOSE_OWN_PIECE);
```

Comment:
centraliser avec RulesFailure ?

Reply:
Ici il y a un contexte en plus: "for your first click, you must choose one of your piece". Plutôt centraliser et perdre le contexte ou pas ?

Reply:
Peut importe, c'est deux phrases communes à plusieurs jeux, des jeux ou le premier clic doit être une de ses pièces, ça manque pas :p

Reply:
Ouais, centraliser disons !